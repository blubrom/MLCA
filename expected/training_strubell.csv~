model,expected energy,estimated energy match,estimated energy base,expected CO_2 e,expected CO_2 e,estimated CO_2 e match,estimated CO_2 e match, estimated CO_2 e base,estimated CO_2 e base
,(kWh),(kWh),(kWh),(kg),(lbs),(kg),(lbs),(kg),(lbs),
Transformer_base,27,38.1,74.8,11.79,26,16.5,36.38,27.7,61.07
Transformer_big,201,267,523,87.09,192,116,255.74,194,427.7
BERT_base,1507,2000,3920,652.17,1438,865,1907,1450,3196.71
NAS,656347,871000,1710000,284018,626155,377000,831143,632000,1393322.63
ELMo,275,404,793,118.84,262,175,385.81,293,645.95