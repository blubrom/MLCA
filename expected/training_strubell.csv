model,expected,estimated,estimated,expected,estimated,estimated,expected,estimated,estimated
,energy,energy match,energy base,CO2e,CO2e match,CO2e base,CO2e,CO2e match,CO2e base
,(kWh),(kWh),(kWh),(kg),(kg),(kg),(lbs),(lbs),(lbs)
Transformer_base,27,38.1,74.8,11.79,16.5,27.7,26,36.38,61.07
Transformer_big,201,267,523,87.09,116,194,192,255.74,427.7
BERT_base,1507,2000,3920,652.17,865,1450,1438,1907,3196.71
NAS,656347,871000,1710000,284018,377000,632000,626155,831143,1393322.63
ELMo,275,404,793,118.84,175,293,262,385.81,645.95
